graylog2-server/src/main/java/org/graylog2/decorators/DecoratorImpl.java

```diff
@@ -30,10 +30,15 @@
 import java.util.Map;
 import java.util.Optional;
 
+import static org.graylog2.database.DbEntity.NO_TITLE;
+import static org.graylog2.shared.security.RestPermissions.DEFLECTOR_READ;
+
 @AutoValue
 @WithBeanGetter
 @JsonAutoDetect
-@CollectionName("decorators")
+@DbEntity(collection = "decorators",
+          titleField = NO_TITLE,
+          readPermission = DEFLECTOR_READ)
```

Comment:
The `DEFLECTOR_READ` permission is related to index management, not to decorators. :slightly_smiling_face:
I guess you meant the `DECORATORS_READ` permission here.

Reply:
Yes! Thanks for spotting this !!!